README.md

```diff
@@ -156,6 +156,17 @@ which fields the ordering is enabled for, pass an array of the field names into
 />
 ```
 
+To have the opposite effect simply use `excludeOrderingBy`:
+
+```JSX
+<AjaxDynamicDataTable
+    rows={this.state.users}
+    excludeOrderingBy={[
```

Comment:
Would it make more sense to go with the convention of `allow` and `disallow`, as the existing prop opposite to this one is `allowOrderingBy`?

Reply:
The naming convention for _not allowing something_ in this package seems to use the word "exclude". For example: [`fieldsToExclude` prop](https://github.com/langleyfoxall/react-dynamic-data-table#excluding-fields) which has been there for ages.

What are your thoughts, @DivineOmega?

Reply:
I think there should be a separation of terminology here.

* For _not allowing something_, we should use `disallow`
* For _not including something_, we should use `exclude`

So, I'd suggest the following.

* `disallowOrderingBy` for _not allowing ordering_
* `fieldsToExclude` for _not including fields_

Thoughts?

Reply:
I feel `disallowOrderBy` also reads a little better than `excludeOrderingBy`, which may not be entirely clear for someone new to the package.

Reply:
@DivineOmega, sure. I like this separation; we should refer to this for future props.